packages/connect-node/src/node-universal-client.ts

```diff
@@ -345,22 +341,20 @@ function h2Request(
     );
     sentinel
       .catch((reason) => {
-        return new Promise<void>((resolve) => {
-          if (stream.closed) {
-            return resolve();
-          }
-          // Node.js http2 streams that are aborted via an AbortSignal close with
-          // an RST_STREAM with code INTERNAL_ERROR.
-          // To comply with the mapping between gRPC and HTTP/2 codes, we need to
-          // close with code CANCEL.
-          // See https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md#errors
-          // See https://www.rfc-editor.org/rfc/rfc7540#section-7
-          if (reason instanceof ConnectError && reason.code == Code.Canceled) {
-            return stream.close(http2.constants.NGHTTP2_CANCEL, resolve);
-          }
-          // For other reasons, INTERNAL_ERROR is the best fit.
-          stream.close(http2.constants.NGHTTP2_INTERNAL_ERROR, resolve);
-        });
+        if (stream.closed) {
+          return;
+        }
+        // Node.js http2 streams that are aborted via an AbortSignal close with
+        // an RST_STREAM with code INTERNAL_ERROR.
+        // To comply with the mapping between gRPC and HTTP/2 codes, we need to
+        // close with code CANCEL.
+        // See https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md#errors
+        // See https://www.rfc-editor.org/rfc/rfc7540#section-7
+        const rstCode =
+          reason instanceof ConnectError && reason.code == Code.Canceled
+            ? H2Code.CANCEL
+            : H2Code.INTERNAL_ERROR;
```

Comment:
No logical changes here, just making use of the added enumeration for error codes.